app/services/requestLogin/index.js

```diff
@@ -0,0 +1,13 @@
+import proxy from 'utils/networkProxy';
+
+// axios instance setting
+export default function requestLogin(email, password) {
```

Comment:
io in genere preferisco usare sempre un argomento con i parametri dentro tipo requestLogin({ email, password })
in questo modo non dipendi + dall'ordine degli argomenti (il primo è email, il secondo è password) che spesso ci porta a rovinare le signature perchè si aggiungono argomenti opzionali, seguiti da argomenti obbligatori.. e ti ritrovi a chiamare "uglySignatureCall(needThis, null, thisToo, null, null, andThisAsWell).

Reply:
di contro usando l'oggetto avresti nicerSignatureCall({ needThis, thisToo, andThisAsWell});